playbook/app/pb_kits/playbook/pb_button/_button.scss

```diff
@@ -1,7 +1,24 @@
 @import "./button_mixins";
 
+$pb_button_sizes: (
+  "sm":   0.75rem,
+  "md":   0.875rem,
+  "lg":   1.125rem,
+);
+
 [class^=pb_button_kit]{
   @include pb_button;
+  // Size =================
+  @each $name, $size in $pb_button_sizes {
+    &[class*=_#{$name}] {
+      font-size: $size;
+      padding: calc(#{$size} / 2) calc(#{$size} * 2.42) !important;
+    }
+  }
+  &[class*=_sm] {
+    min-height: 0;
```

Comment:
Curious on this one too. Once the button is sized down to "sm", does this keep the box model consistent?

Reply:
Since sm size is only 12px min-height needs to be set to 0 only for this size.